### ISSUE TITLE
Issues related to input mapping

### ISSUE THREAD

**Issue by greenpixels** [limbonaut/limbo_console#1]
Heyho! Just added your project as a submodule and trying it out. Really cool!
Just two things I noticed:

1) Does not show in InputMap

For some reason, the InputMap does not show limbo_console_toggle in the input-map overview despite it definitley being set in `project.godot`. Not sure whether that is something you have any reign over. 
![grafik](https://github.com/user-attachments/assets/5f32611b-8f85-42df-b8d4-c76bd1cb2d79)
![grafik](https://github.com/user-attachments/assets/5e11125c-68fd-4c13-9922-a90be3a94af4)

2) Documentaiton incorrect and QUOTE_LEFT doesn't work (QWERTY)

Also, in the docs it says to press `~`, but on-init the key-code is set to `96` which is the QUOTE_LEFT key.
![grafik](https://github.com/user-attachments/assets/91076e94-b71b-40fc-b52c-7d315afccd0d)
Sadly, the QUOTE_LEFT key does not work for me. It does not register for some reason. I wanted to change the key in my input_map, but as described earlier - it does not show up haha.

I've manually set the key code to `F10` in my `project.godot` file.

**Comment by limbonaut** [comment]
Right, it is indeed set to QUOTE_LEFT by default: https://github.com/limbonaut/limbo_console/blob/aa286e83053dea2e8484e2a56ba18aea4a6b7700/plugin.gd#L19

> For some reason, the InputMap does not show limbo_console_toggle in the input-map overview

Have you tried restarting Godot after enabling the plugin in the project settings? I should probably mention it in the README. Just tested on an empty project with a submodule, and it shows up as expected after I restarted the editor.

**Comment by Calinou** [comment]
This is due to a bug in the editor: https://github.com/godotengine/godot/issues/25865

**Comment by limbonaut** [comment]
Thanks for the link, @Calinou !

For now, I mentioned in the README that the project reload is needed after you enable the plugin. Keeping this open to track this issue.
